src/iterative_sorting/iterative_sorting.py

```diff
@@ -6,21 +6,28 @@ def selection_sort( arr ):
         smallest_index = cur_index
         # TO-DO: find next smallest element
         # (hint, can do in 3 loc) 
-             
-
-
-
+        for j in range(cur_index, len(arr)):
+            if arr[j] < arr[smallest_index]:
+                smallest_index = j
         # TO-DO: swap
-
-
-
+        cur_value = arr[smallest_index]
+        arr[smallest_index] = arr[cur_index]
+        arr[cur_index] = cur_value
 
     return arr
 
 
 # TO-DO:  implement the Bubble Sort function below
 def bubble_sort( arr ):
-
+    swap = True
+    while swap:
+        swap = False
+        for i in range(0, len(arr) - 1):
+            if arr[i] > arr[i + 1]:
+                cur_value = arr[i]
```

Comment:
You can also swap the two elements in the list by doing `arr[i], arr[i + 1] = arr[i + 1], arr[i]`. With this trick you don't need to create a temp variable.